recipes-core/packagegroups/packagefeed-ni-extra.bb

```diff
@@ -207,7 +209,7 @@ RDEPENDS:${PN} += "\
 	systemtap \
 "
 
-# openembedded-gore/meta/recipes-multimedia
+# openembedded-core/meta/recipes-multimedia
```

Comment:
Ideally we should make unrelated changes in another commit so it could be cherry-picked into other branches (like kirkstone) or enable us to easily revert subset of commits from a PR if needed.